main/lsp/tools/make_lsp_types.cc

```diff
@@ -1244,11 +1244,7 @@ void makeLSPTypes(vector<shared_ptr<JSONClassType>> &enumTypes, vector<shared_pt
                    classTypes);
 
     // Empty object.
-    auto InitializedParams = makeObject("InitializedParams", {}, classTypes,
```

Comment:
The preprocessor used to populate these parameters with `LSPFileUpdates`. Not anymore!